test/integrations/testUtils.ts

```diff
@@ -25,7 +25,9 @@ export const getTestDataFilePaths = (dirPath: string, opts: OptionValues): strin
   const globPattern = join(dirPath, '**', 'data.ts');
   let testFilePaths = globSync(globPattern);
   if (opts.destination) {
-    testFilePaths = testFilePaths.filter((testFile) => testFile.includes(opts.destination));
+    testFilePaths = testFilePaths.filter((testFile) =>
+      testFile.includes(`/destinations/${opts.destination}/`),
```

Comment:
why this directory update ?

Reply:
When running test cases by giving the destination flag
`includes` matches both ga4 and ga4_v2 for `ga4`

Hence fixed it